Packs/Zafran/Integrations/Zafran/Zafran.yml

```
@@ -0,0 +1,136 @@
+    description: Update on mitigations performed
+    arguments:
+    - name: mitigationstatus_external_ticket_id
+      description: 'External service ticket ID.'
+      required: false
+      isArray: false
+    - name: mitigationstatus_external_ticket_url
+      description: 'External service ticket link.'
+      required: false
+      isArray: false
+    - name: mitigationstatus_id
+      description: 'Mitigation ID.'
+      required: false
+      isArray: false
+    - name: mitigationstatus_state
+      description: 'Mitigation status     new - New mitigation     pending_approval - Waiting for mitigation approval     rejected - Mitigative action was rejected.     in_progress - Mitigation approved and in progress.     completed - Mitigation applied successfully.'
+      required: false
+      isArray: false
+      auto: PREDEFINED
+      predefined:
+      - new
+      - pending_approval
+      - rejected
+      - in_progress
+      - completed
+    outputs:
+    - type: Number
+      contextPath: Zafran.MitigationsPerformedResponse.internal_status_code
+      description: 'Internal status code.'
+    - type: String
+      contextPath: Zafran.MitigationsPerformedResponse.message
+      description: 'Error message.'
+  - name: zafran-mitigations-export
+    description: Export recommended mitigations
+    arguments:
+    - name: filter_
+      description: ZQL filter
+      required: false
+      isArray: false
+    outputs:
+    - type: Number
+      contextPath: Zafran.UpstreamMitigation.assets_count
+      description: 'Assets count.'
+    - type: String
+      contextPath: Zafran.UpstreamMitigation.control_product
+      description: 'Control Product.'
+    - type: Number
+      contextPath: Zafran.UpstreamMitigation.exposure
+      description: 'Exposure in days.'
+    - type: String
+      contextPath: Zafran.UpstreamMitigation.id
+      description: 'Zafran mitigation unique id.'
+    - type: String
+      contextPath: Zafran.UpstreamMitigation.mitigation_type
+      description: 'Mitigation type.'
+    - type: String
+      contextPath: Zafran.UpstreamMitigation.recommendation
+      description: 'Recommendation.'
+    - type: String
+      contextPath: Zafran.UpstreamMitigation.title
+      description: 'Title.'
+    - type: Number
+      contextPath: Zafran.UpstreamMitigation.vulnerabilities_count
+      description: 'Vulnerabilities count.'
+    - type: Number
+      contextPath: Zafran.UpstreamMitigation.internal_status_code
+      description: 'Internal status code.'
+    - type: String
+      contextPath: Zafran.UpstreamMitigation.message
+      description: 'Error message'
+  - name: zafran-mitigations-performed
+    description: Update on mitigations performed
+    arguments:
+    - name: mitigationsstatus_mitigation_id
+      description: 'Mitigation ID.'
+      required: false
+      isArray: false
+    - name: mitigationsstatus_mitigation_ids
+      description: 'Mitigation IDs.'
+      required: false
+      isArray: true
```

```suggestion
    - name: mitigationsstatus_mitigation_ids
      description: 'Mitigation IDs.'
      required: false
      isArray: true
```
why do we need both of them? whats the difference between those arguments?